Microsoft.Identity.Web/README.md

```diff
@@ -75,16 +83,16 @@ public class Startup
   public void ConfigureServices(IServiceCollection services)
   {
    ...
-   services.AddMicrosoftIdentityPlatformAuthentication(Configuration)
-           .AddMsal(new string[] { scopesToRequest })
+   services.AddSignIn(Configuration)
+           .AddWebAppCallsProtectedWebApi(new string[] { scopesToRequest })
```

Comment:
>AddWebAppCallsProtectedWebApi [](start = 12, length = 29)

Another suggestion :). Also let’s hold this change till we have more useful breaking changes to ship. Aggravating developers who have adopted this library for a mere name change will not earn more goodwill and it seems we need more of that to be able to ship this library 😊